src/containers/workExperiences/workExperiences.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./workExperiences.css";
+import ExperienceCard from "../../components/experienceCard/ExperienceCard";
+import { workExperiences } from "../../portfolio";
+import { Fade } from "react-reveal";
+
+
+export default function Experiences() {
```

Comment:
Let's keep the export name consistent to the file name for standards and less confusion.

Also rename file to `workExperience` and update wherever required in other files too.